h/services/group_create.py

```diff
@@ -126,7 +126,9 @@ def _create(self, name, userid, type_flags, scopes, **kwargs):
         # self.publish() or `return group`.
         self.db.flush()
 
-        group.members.append(group.creator)
+        self.db.add(
+            GroupMembership(user_id=creator.id, group_id=group.id, roles=["owner"])
+        )
```

Comment:
I don't know if there's any way to pass through the `roles=["owners"]` when adding the membership via the `group.members` relationship? So constructing the `GroupMembership` object directly instead.

Reply:
To be honest I prefer this style.